### PR TITLE
tests: initial implementation

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

**Comment by andy5995** [comment]
I'll add a README.md to the tests directory with some instructions.

**Review by winrid — APPROVED** [review]
Nice! Tests should definitely not be mandatory, but it is nice to be able to write them for core functionality!

**Comment by andy5995** [comment]
Thanks for reviewing! I've made a few changes and I think it's ready now.

File: src/Lib/package.cpp [hunk 1]
```diff
@@ -31,3 +33,19 @@ const std::string Package::GetName(void) { return std::string("NetPanzer"); }
...
+
+int main(void)
+{
+  int len = Package::GetVersion().length();
+  assert(len >= 5 && len < 10);
+  assert(Package::GetName().compare("NetPanzer") == 0);
+
+  return 0;
+}
+
```
Comment (winrid) [inline src/Lib/package.cpp:50]: are you saying we don't need to switch to Rust to have tests in the same file as the code!?! :)

File: .cirrus.yml [hunk 2]
```diff
@@ -14,6 +14,8 @@ freebsd_task:
     - meson setup _build
   build_script:
     - cd _build && ninja
+  test_script:
+    - cd _build && meson test
```
Comment (winrid) [inline .cirrus.yml:18]: curious, why underscore?

**Review by winrid — APPROVED** [review]

File: src/Lib/package.cpp (hunk 1 above)
Comment (andy5995) [reply src/Lib/package.cpp:50]: lol. Nope! I try to think of everything! But you're observant for sure, I did get the idea from rust. :)

**Pull request merged**